app/build.gradle

```diff
@@ -148,6 +148,8 @@ dependencies {
     //Glide
     implementation 'com.github.bumptech.glide:glide:4.12.0'
     annotationProcessor 'com.github.bumptech.glide:compiler:4.12.0'
+
+    implementation("io.github.coordinates2country:coordinates2country-android:1.2") {  exclude group: 'com.google.android', module: 'android' }
```

Comment:
Thanks for figuring out a way to make this work!
By any chance, would you have any advice on fixing the library itself so that the `{ ... }` part becomes unecessary? :-)

Reply:
I think the transitive Android dependency in `build.gradle` might be responsible, So replacing it could be a solution.